platform/ontology/api.go

```diff
@@ -16,6 +16,8 @@ var client = Client{
 }
 
 const GovernanceContract = "AFmseVrdL9f9oyCzZefL9tG6UbviEH9ugK"
+const ONTAssetName = "ont"
+const ONGAssetName = "ong"
```

Comment:
check this file, that's how you specify consts: https://golang.org/src/math/const.go